fire_atlas_editor/lib/utils/select_file.dart

```diff
@@ -1,18 +1,18 @@
 import 'dart:html';
 
 void selectFile(void Function(String) callback) {
-  InputElement uploadInput = FileUploadInputElement();
+  final uploadInput = FileUploadInputElement();
   uploadInput.click();
 
   uploadInput.onChange.listen((e) {
     // read file content as dataURL
     final files = uploadInput.files;
```

Comment:
if you do `?? []` here your error handling below might be cleaner

Reply:
Changed